src/PermissionManager.php

```diff
@@ -67,14 +68,9 @@ public function getPermissionList($entity_type_id, $bundle_id) {
   /**
    * {@inheritdoc}
    */
-  public function generateCrudPermissionList($group_content_entity_type_id, $group_content_bundle_id) {
+  public function generateEntityOperationPermissionList($group_content_entity_type_id, $group_content_bundle_id) {
     $permissions = [];
 
-    // Check if the bundle is a group content type.
-    if (!Og::isGroupContent($group_content_entity_type_id, $group_content_bundle_id)) {
-      return [];
-    }
```

Comment:
Now that we get trusted data directly from the `GroupManager` we don't need to do check this here any more.